sirepo/job_driver/sbatch.py

```diff
@@ -211,12 +211,13 @@ def _raise_sbatch_login_srexception(self, reason, msg):
         raise util.SRException(
             'sbatchLogin',
             PKDict(
+                host=self.cfg.host,
                 isModal=True,
+                nonJobExc=True,
                 reason=reason,
+                report=msg.computeModel,
```

Comment:
Since `report` is new, let's leave it `computeModel=msg.computeModel`. It's not a "report" if it is a computeModel.

Reply:
https://github.com/radiasoft/sirepo/pull/2523/commits/2c505a1eca579dafbef6921f946a8bc6a642cdc8